src/bin_to_hex.erl

```diff
@@ -24,6 +24,14 @@ bin_to_hex_(<<A:8, B:8, C:8, D:8, E:8, F:8, G:8, H:8, Rest/binary>>, Acc) ->
     <<Acc/binary,
       ?H(A), ?H(B), ?H(C), ?H(D), ?H(E), ?H(F), ?H(G), ?H(H)>>).
 
+-spec hex_to_bin(binary()) -> <<_:_*1>>.
+hex_to_bin(Bin) when is_binary(Bin) ->
```

Comment:
would compiling THIS function to native improve performance?

Reply:
No idea, but if it belongs in any single module it's this one.

Reply:
so, I suggest microbenchmark it.
bin_to_hex is optimized for native, but not all code runs good using hipe.

Reply:
Microbenchmark or profiling should be a separate task.